build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/DefaultBuildCoordinator.java

```diff
@@ -147,8 +151,8 @@ public BuildSetTask build(BuildConfigurationSet buildConfigurationSet, User user
                 rebuildAll,
                 getBuildStatusChangedEventNotifier(),
                 () -> datastoreAdapter.getNextBuildRecordId());
-        BuildSetStatusChangedEvent event = buildSetTask.setStatus(BuildSetStatus.NEW);
-        buildSetStatusChangedEventNotifier.fire(event);
+        notifyBuildSetStatusChanged(buildSetTask, BuildSetStatus.NEW);
```

Comment:
it might be more readable if notifyBuildSetStatusChanged was called e.g. updateBuildSetStatus and contained setStatus